app/views/shared/_error_messages.html.erb

```diff
@@ -1,7 +1,7 @@
 <% if @user.errors.any? %>
   <div id="error_explanation">
     <div class="alert alert-danger">
-      <%= t ".form_error_warning", errors_count: @user.errors.count %>
+      <%= t ".form_error_warning", errors_count: pluralize(@user.errors.count, "error") %>
```

Comment:
1. you should using `size` instead of `count`
Can you tell me what are different of `size`, `length` and `count`
2. Using params I18n `@user.errors.count, "error"`
`number_error: " %{number} error"`

Reply:
@VuThiTranVan count is counting records in db, length is in array, size choose the faster way to do.